README.md

```diff
@@ -264,6 +264,16 @@ It is possible to use an HTML form for uploading files. To do so, you need to up
 
 For an example, see [examples/html_form.toml](./examples/html_form.toml)
 
+#### Retrieving list of files
```

Comment:
As @tessus pointed out, this should definitely be guarded behind an authentication token. Let's implement that and also make a note in README.md as well.

Also, see #28